content/tutorial/tutorial.md

```diff
@@ -1170,41 +1177,41 @@ We will also replace reading `this.state.history` with `this.state.history.slice
   }
 ```
 
-Finally, we will modify the Game component's `render` method from always rendering the last move to rendering the currently selected move according to `stepNumber`:
+अंत में, हम Game कौम्पोनॅन्ट के `render` मेथड को हमेशा अंतिम चरण प्रदान करने से लेकर वर्तमान में चुने गए कदम को `stepNumber` के अनुसार रेंडर करेंगे।
 
 ```javascript{3}
   render() {
     const history = this.state.history;
     const current = history[this.state.stepNumber];
     const winner = calculateWinner(current.squares);
 
-    // the rest has not changed
+    // और इसके आगे कोई चाल बाकि नहीं है
 ```
 
-If we click on any step in the game's history, the tic-tac-toe board should immediately update to show what the board looked like after that step occurred.
+यदि हम गेम के history के किसी भी चरण पर क्लिक करते हैं, तो टिक-टैक-टो Board को तुरंत यह दिखाने के लिए अपडेट करना चाहिए कि Board उस चरण के बाद कैसा दिखता था।
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/gWWZgR?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/gWWZgR?editors=0010)**
 
-### Wrapping Up {#wrapping-up}
+### समाप्त करें {#wrapping-up}
 
-Congratulations! You've created a tic-tac-toe game that:
+बधाई हो! आपने एक टिक-टैक-टो गेम बनाया है:
 
-* Lets you play tic-tac-toe,
-* Indicates when a player has won the game,
-* Stores a game's history as a game progresses,
-* Allows players to review a game's history and see previous versions of a game's board.
+* अब टिक-टैक-टो खेलते हैं,
+* इंगित करता है कि जब किसी खिलाड़ी ने गेम जीता है,
+* एक खेल के रूप में एक खेल के इतिहास को स्टोर करता है,
+* खिलाड़ियों को गेम के इतिहास की समीक्षा करने और गेम के बोर्ड के पिछले संस्करणों को देखने की अनुमति देता है।
```

Comment:
revisit bullets once?

Reply:
```
* जो आपको टिक-टैक-टो खेलने देता है 
* संकेत करता है जब कोई खिलाड़ी गेम जीतता है,
* खेल के प्रोग्रेस को गेम की हिस्ट्री में रखता है,
* प्लेयर्स को गेम के हिस्ट्री को रिव्यु करने और गेम के बोर्ड के पिछले वेर्सिओंस को देखने की अनुमति देता है।
```

Reply:
वेर्सिओंस could have a better spelling, rest is good

Reply:
> वेर्सिओंस could have a better spelling, rest is good

`वर्शनस`